mailet/standard/pom.xml

```diff
@@ -56,6 +56,10 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>james-server-util-java8</artifactId>
         </dependency>
+        <dependency>
```

Comment:
No, we should not have that dependancy! 

I should be able to use these components without depending on James internals.

Let's not break this for now...

Reply:
much better in server, thank you